hloc/localize_sfm.py

```diff
@@ -47,15 +48,17 @@ def do_covisibility_clustering(frame_ids, all_images, points3D):
 
 def pose_from_cluster(qname, qinfo, db_ids, db_images, points3D,
                       feature_file, match_file, thresh):
-    kpq = feature_file[qname]['keypoints'].__array__()
+    if platform.system() == 'Windows':
+        kpq = feature_file[qname.replace('/', '\\')]['keypoints'].__array__()
+    else:
+        kpq = feature_file[qname]['keypoints'].__array__()
```

Comment:
Do we really need to rename all query and reference paths in all these files? I would think that there are simpler options:
1. Update [`parse_retrieval`](https://github.com/cvg/Hierarchical-Localization/blob/master/hloc/utils/parsers.py#L28) to convert query and reference paths from UNIX to Windows
2. Update the [`ImageDataset`](https://github.com/cvg/Hierarchical-Localization/blob/master/hloc/extract_features.py#L110) to always return Posix paths with `path.as_posix()` instead of `str(path)`.

I would favor 2. but don't know if COLMAP parses images as Windows or Posix paths.

Reply:
I will test solution 2 with "pipeline_Aachen.ipynb" script. It may take two days on my Windows system.